### DISCUSSION TITLE
Add the ovdet algorithm to the mmdet projects. 给 mmdet projects 支持 ovdet 开放词汇算法

### DISCUSSION THREAD

**Discussion by OpenMMLab-Assistant-004** [open-mmlab/OpenMMLabCamp#628] [MMDetection]
# Add the ovdet algorithm to the mmdet projects. 
 给 mmdet projects 支持 ovdet 开放词汇算法 

Support training and inference of the ovdet algorithm in the projects of mmdet, with reference to the open source code [ovdet](https://github.com/wusize/ovdet)

参考基于 mmdet 的官方开源代码 [ovdet](https://github.com/wusize/ovdet)，在 mmdet projects 中支持改算法训练和推理

## Task Information

**Repository**：MMDetection

**Difficulty**：Hard

**Points**：80

**Estimated Time Consumption**：35 days

## Task Status

**Claimant's Github ID**：None

**Claim Time**：2023年09月11日

**PR Link**：None

## Points to Note

To claim the task, please reply `Claim the mission` in the comment section. After claiming the task, please add our assistant bot called MeowMewo on WeChat at: `openmmlabwx`.

After completing the task, please submit a PR with the title starting with [MMSIG]. After submission, please reply with your PR link in the comments section of this Discussion. We will review your work as soon as possible.

在评论区回复 `Claim the mission` 领取任务，领取任务后请及时添加小助手喵喵微信: `openmmlabwx`

完成任务后提交相关链接（如是PR，标题请以[MMSIG]开头）回复到本Disucssion评论区，我们会第一时间审核您的贡献。

**Comment by Shengshenlan** [comment]
Claim the mission

**Comment by Shengshenlan** [comment]
https://github.com/open-mmlab/mmdetection/pull/11217
